nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,71 @@
+ 
+NOTE: No matter what deployment options are chosen, there is one general rule that should always apply: **All transport data (input, audit and error queues) should reside in a single SQL Server catalog**. Multi-instance/multi-catalog deployment topology is deprecated in version 3 of the SQL Server transport and will be removed in version 4.
+
+### Transactionality
+SQL Server Transport supports all [transaction modes](/nservicebus/transports/transactions.md) in particular `TransactionScope` which enables `exactly once` message processing with usage of distributed transactions. However when transport, persistence and business data are all stored in a single SQL Server catalog it is possible to achieve `exactly-once` message delivery without need for distributed transactions. [SQL Server native integration sample](/samples/sqltransport/native-integration/) contains more information on such setup.
+
+NOTE: `Exactly once` message processing without distributed transactions can be achieved with any transport (and SQL Server Transport in particular) using [Outbox](/nservicebus/outbox/). It requires business and persistence data to share the storage mechanism but does not put any requirements on transport data storage.
+
+### Security 
+Security considerations for SQL Server Transport should follow the principle of least privilege. For that each endpoint should use dedicated SQL Server principal with `SELECT` and `DELETE` permissions on it's input queues as well as `INSERT` permissions on other endpoint's input queues, audit queue and error queue. 
+
+Multi-schema configuration can be used to ease the maintenance of security configuration because each endpoint may own a number of queues.
+
+### Monitoring
+All deployment options for SQL Server Transport described in this section are supported by Service Control.
+
+## Sample usage scenarios
+### Tiny
+
+The SQL Server transport is an ideal choice for extending an existing web application with asynchronous processing capabilities as an alternative for tedious batch jobs that tend to quickly get out of synch with the main codebase. Assuming the application already uses SQL Server as a data store, this scenario does not require any additional infrastructure.
+
+The queue tables are hosted in the same catalog as business data and the NServiceBus runtime can be hosted in the web worker. In some cases there might be a need for a separate process for hosting NServiceBus. Because system consists of a single logical service or bounded context, there is usually no need to create separate schemas for the queues.
+
+### Small
+
+The SQL Server transport is a good choice for a pilot project to prove feasibility of NServiceBus in a given organization as well as for a small, well-defined green field application. It usually requires nothing more than a single shared SQL Server instance.
```

Comment:
why is "tiny" a real system. but small is a pilot? i would prefer to see pilot mentioned in its own section